lib/wcc/contentful/store/cdn_adapter.rb

```diff
@@ -67,6 +67,12 @@ def nested_conditions(field, conditions, context)
         end
       end
 
+      Base::Query::OPERATORS.each do |op|
```

Comment:
is this necessary?

Reply:
Not for our current use cases, but it makes it consistent with the other stores that provide a `.eq` method.  As I was writing the documentation for that layer, it seemed to me that `apply_operator` is not great from an API design perspective and the convenience methods are useful.

Reply:
Understandable.. the `apply_operator` method was really a way to get away from calling `send`.